mirascope/core/base/_call_factory.py

```diff
@@ -310,7 +490,7 @@ def base_call(
                 json_mode=json_mode,
                 client=client,
                 call_params=call_params,
-            )  # pyright: ignore [reportReturnType]
+            )  # pyright: ignore [reportReturnType, reportCallIssue]
```

Comment:
I think I resolved all types. But, I can't resolve the error.
```bahs
  /Users/koudai/PycharmProjects/mirascope/mirascope/core/base/_call_factory.py:480:20 - error: No overloads for "decorator" match the provided arguments (reportCallIssue)
1 error, 0 warnings, 0 informations 
```

Reply:
ok, that's fine for now, we can revisit this later if necessary